rclrs/README.md

```diff
@@ -0,0 +1,5 @@
+# ROS 2 client library for Rust
+
+ROS 2 is a popular robotics framework, used in a variety of fields (self-driving cars, drones, humanoid robots, etc.). `rclrs` is a library for writing robotics applications in Rust that integrate with ROS 2.
```

Comment:
Can you add opensource? i.e. "popular opensource robotics framework" Thanks.

Reply:
Good point, done (though I used "open source" instead of "opensource", I think that's the standard spelling).